### PR TITLE
LPScrollOperation: scroll with adjustedContentInset when available

### PR DESCRIPTION
**Pull request by jmoody**
### Motivation

Fixes:

* UITest: Timeout error on iOS Scrolling on local iOS11 device [VSTS](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/29352)

### Test

- [x] Reproduce on local device
- [x] Demonstrate fix on local device

### CODE REVIEW

**Comment by jmoody** [comment]
@polyakrecords You can review, but this is not ready for merging.

File: XCTest/Tests/Operations/LPScrollOperationTest.m [hunk 1]
```diff
@@ -3,6 +3,8 @@
 #endif
 
 #import "LPScrollOperation.h"
+#import "LPInvoker.h"
```
Comment (owenniblock) [inline XCTest/Tests/Operations/LPScrollOperationTest.m]: I can't see any code changes to this file? Not sure if these additional `import` statements are necessary?

Comment (jmoody) [reply XCTest/Tests/Operations/LPScrollOperationTest.m]: We are using LPInvoker + LPInvocationResult to call`adjustedContentInset` in a memory-safe way.

File: bin/test/xctest.rb [hunk 2]
```diff
@@ -25,13 +25,7 @@
...
 sim_udid = default_sim.udid
 
-xcode = RunLoop::Xcode.new
-
-if xcode.version_gte_7?
-  warnings_as_errors="GCC_TREAT_WARNINGS_AS_ERRORS=YES"
-else
-  warnings_as_errors="GCC_TREAT_WARNINGS_AS_ERRORS=NO"
-end
+warnings_as_errors=
```
Comment (jmoody) [inline bin/test/xctest.rb]: This needs to be deleted

**Comment by jmoody** [comment]
@polyakrecords This is ready for review and merge.

**Pull request merged**